packages/controllers/src/snaps/SnapController.ts

```diff
@@ -14,7 +14,7 @@ import { SerializedEthereumRpcError } from 'eth-rpc-errors/dist/classes';
 import type { Patch } from 'immer';
 import { Json } from 'json-rpc-engine';
 import { nanoid } from 'nanoid';
-import isValidSemver from 'semver/functions/valid';
+import semver from 'semver';
```

Comment:
Would we get a bundle size benefit by importing only the used `semver` functions or can they be tree-shaken when importing everything?